tests/functional/t/too/too_many_ancestors.txt

```diff
@@ -1,2 +1,2 @@
-too-many-ancestors:20:0:Iiii:Too many ancestors (9/7)
-too-many-ancestors:23:0:Jjjj:Too many ancestors (10/7)
+too-many-ancestors:21:0:Iiii:Too many ancestors (8/7)
```

Comment:
This actually fixed that test case too, the correct number IS 8 !

Reply:
Depends if you con't `object`, but it's certainly not false.